.github/workflows/coq-action.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    # the OS must be GNU/Linux to be able to use the docker-coq-action
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - 'coqorg/coq:dev'
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - uses: coq-community/docker-coq-action@v1
+        with:
+          opam_file: 'coq-paramcoq.opam'
```

Comment:
IIUC this will compile and install the plugin but no longer run the test suite?

Reply:
See https://github.com/coq/bignums/pull/51#discussion_r528340469

Reply:
Right, I didn't realize. I will use a custom script similar to the one in bignums for now, but we should definitely think at how to extend the template so that this is supported. Do you also need scheduled pipelines?

Reply:
Not really. This is in Coq's CI, I'll just need to rerun CI manually one day after merging each overlay, but they are rare enough that it's not really cumbersome.